modules/apps/foundation/frontend-theme/responsive-boat-theme/src/js/main.js

```diff
@@ -0,0 +1,39 @@
+
+// This function gets loaded when everything, including the portlets, is on the page
+
+Liferay.on('allPortletsReady', myPageJS);
+
+// Function Containing All of My Event Handlers
+
+function myPageJS() {
+	let nav = AUI.$('#navigation');
+	let navBtnDiv = AUI.$('#navigation div.nav-btn');
```

Comment:
Since you have ```nav``` already set as a variable, can we just leverage that variable instead of doing another DOM lookup?